README.md

```diff
@@ -240,6 +240,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
+| v0.46.0                | v1.21 to v1.23       | 1.6.1                |
```

Comment:
This needs to be changed to 1.19 to 1.23

Reply:
Done